### PR TITLE
Add date show version

### PR DESCRIPTION
**Pull request by EdenGri**
#### What I did

Add current date attribute to the "show version" output that includes the current date and hour on the switch.
remove the hour from the uptime attribute.

#### How I did it

In sonic-utilities, in the "version" subcommand, print of the current date will be added using "datetime" from the "datetime" library.

#### How to verify it

Execute "show version" command and verify "Date" row under the "Uptime" row.

#### Previous command output (if the output of a command-line utility has changed)

admin@sonic:~$ show version
SONiC Software Version: SONiC.HEAD.32-https://github.com/EdenGri/sonic-utilities/commit/21ea29a23f1f1b4a3ce294e6cbdaee4388047231
Distribution: Debian 9.8
Kernel: 4.9.0-8-amd64
Build commit: 21ea29a
Build date: Fri Mar 22 01:55:48 UTC 2019
Built by: johnar@jenkins-worker-4

Platform: x86_64-mlnx_msn2700-r0
HwSKU: Mellanox-SN2700
ASIC: mellanox
ASIC Count: 1
Serial Number: MT1822K07815
Model Number: MSN2700-CS2FO
Hardware Rev: A1
Uptime: 14:40:15 up 3 min, 1 user, load average: 1.26, 1.45, 0.66
...

#### New command output (if the output of a command-line utility has changed)

admin@sonic:~$ show version
SONiC Software Version: SONiC.HEAD.32-https://github.com/EdenGri/sonic-utilities/commit/21ea29a23f1f1b4a3ce294e6cbdaee4388047231
Distribution: Debian 9.8
Kernel: 4.9.0-8-amd64
Build commit: 21ea29a
Build date: Fri Mar 22 01:55:48 UTC 2019
Built by: johnar@jenkins-worker-4

Platform: x86_64-mlnx_msn2700-r0
HwSKU: Mellanox-SN2700
ASIC: mellanox
ASIC Count: 1
Serial Number: MT1822K07815
Model Number: MSN2700-CS2FO
Hardware Rev: A1
Uptime: up 3 min, 1 user, load average: 1.26, 1.45, 0.66
Date: Tue 22 Feb 2022 14:40:15
...

### CODE REVIEW

**Review by noaOrMlnx — APPROVED** [review]

File: show/main.py [hunk 1]
```diff
@@ -1132,7 +1141,7 @@ def techsupport(since, global_timeout, cmd_timeout, verbose, allow_process_stop,
 
     if since:
         cmd += " -s '{}'".format(since)
-    
+
```
Comment (dprital) [inline show/main.py:1144]: Please remove this change

File: show/main.py [hunk 2]
```diff
@@ -1237,7 +1246,7 @@ def snmp(ctx, db):
 
 # ("show runningconfiguration snmp community")
 @snmp.command('community')
-@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL, 
+@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL,
```
Comment (dprital) [inline show/main.py:1249]: Please remove this change

File: show/main.py [hunk 3]
```diff
@@ -1258,7 +1267,7 @@ def community(db, json_output):
 
 # ("show runningconfiguration snmp contact")
 @snmp.command('contact')
-@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL, 
+@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL,
```
Comment (dprital) [inline show/main.py:1270]: Please remove this change

File: show/main.py [hunk 4]
```diff
@@ -1286,7 +1295,7 @@ def contact(db, json_output):
 
 # ("show runningconfiguration snmp location")
 @snmp.command('location')
-@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL, 
+@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL,
```
Comment (dprital) [inline show/main.py:1298]: Please remove this change

File: show/main.py [hunk 5]
```diff
@@ -1313,13 +1322,13 @@ def location(db, json_output):
 
 # ("show runningconfiguration snmp user")
 @snmp.command('user')
-@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL, 
+@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL,
```
Comment (dprital) [inline show/main.py:1325]: Please remove this change

File: show/main.py [hunk 6]
```diff
@@ -1313,13 +1322,13 @@ def location(db, json_output):
...
               help="Display the output in JSON format")
 @clicommon.pass_db
 def users(db, json_output):
     """show SNMP running configuration user"""
     snmp_users = db.cfgdb.get_table('SNMP_USER')
-    snmp_user_header = ['User', "Permission Type", "Type", "Auth Type", "Auth Password", "Encryption Type", 
+    snmp_user_header = ['User', "Permission Type", "Type", "Auth Type", "Auth Password", "Encryption Type",
```
Comment (dprital) [inline show/main.py:1331]: Please remove this change

File: show/main.py [hunk 7]
```diff
@@ -1332,7 +1341,7 @@ def users(db, json_output):
             snmp_user_encryption_type = snmp_users[snmp_user].get('SNMP_USER_ENCRYPTION_TYPE', 'Null')
             snmp_user_encryption_password = snmp_users[snmp_user].get('SNMP_USER_ENCRYPTION_PASSWORD', 'Null')
             snmp_user_type = snmp_users[snmp_user].get('SNMP_USER_TYPE', 'Null')
-            snmp_user_body.append([snmp_user, snmp_user_permissions_type, snmp_user_type, snmp_user_auth_type, 
+            snmp_user_body.append([snmp_user, snmp_user_permissions_type, snmp_user_type, snmp_user_auth_type,
```
Comment (dprital) [inline show/main.py:1344]: Please remove this change

File: show/main.py [hunk 8]
```diff
@@ -1349,7 +1358,7 @@ def show_run_snmp(db, ctx):
     snmp_contact_body = []
     snmp_comm_header = ["Community String", "Community Type"]
     snmp_comm_body = []
-    snmp_user_header = ['User', "Permission Type", "Type", "Auth Type", "Auth Password", "Encryption Type", 
+    snmp_user_header = ['User', "Permission Type", "Type", "Auth Type", "Auth Password", "Encryption Type",
```
Comment (dprital) [inline show/main.py:1361]: Please remove this change

File: show/main.py [hunk 9]
```diff
@@ -1383,7 +1392,7 @@ def show_run_snmp(db, ctx):
         snmp_user_encryption_type = snmp_users[snmp_user].get('SNMP_USER_ENCRYPTION_TYPE', 'Null')
         snmp_user_encryption_password = snmp_users[snmp_user].get('SNMP_USER_ENCRYPTION_PASSWORD', 'Null')
         snmp_user_type = snmp_users[snmp_user].get('SNMP_USER_TYPE', 'Null')
-        snmp_user_body.append([snmp_user, snmp_user_permissions_type, snmp_user_type, snmp_user_auth_type, 
+        snmp_user_body.append([snmp_user, snmp_user_permissions_type, snmp_user_type, snmp_user_auth_type,
```
Comment (dprital) [inline show/main.py:1395]: Please remove this change

**Pull request closed without merging**